code/modules/power/singularity/emitter.dm

```diff
@@ -308,3 +335,128 @@
 		emagged = 1
 		if(user)
 			user.visible_message("[user.name] emags the [src.name].","<span class='notice'>You short out the lock.</span>")
+
+
+/obj/machinery/power/emitter/prototype
+	name = "Prototype Emitter"
+	icon = 'icons/obj/turrets.dmi'
+	icon_state = "protoemitter"
+	can_buckle = TRUE
+	buckle_lying = 0
+	var/view_range = 12
+	var/datum/riding/riding_datum = null

```

Comment:
Yea I need to remove that, it was a brief experiment to see if it was easier to make these into non-mobile vehicles.